### PR TITLE
Add deployment for Custom Metrics - Stackdriver Adapter using beta API

### PR DESCRIPTION
**Pull request by kawych**

### CODE REVIEW

File: custom-metrics-stackdriver-adapter/adapter-beta.yaml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+subjects:
+- kind: ServiceAccount
+  name: custom-metrics-stackdriver-adapter
+  namespace: custom-metrics
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: custom-metrics-auth-reader
+  namespace: kube-system
```
Comment (piosz) [inline custom-metrics-stackdriver-adapter/adapter-beta.yaml]: should be custom-metrics?

File: custom-metrics-stackdriver-adapter/adapter-beta.yaml [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  template:
+    metadata:
+      labels:
+        run: custom-metrics-stackdriver-adapter
+        k8s-app: custom-metrics-stackdriver-adapter
+        kubernetes.io/cluster-service: "true"
+    spec:
+      serviceAccountName: custom-metrics-stackdriver-adapter
+      containers:
+      - image: gcr.io/kawych-test/custom-metrics-stackdriver-adapter:v0.2.0
```
Comment (piosz) [inline custom-metrics-stackdriver-adapter/adapter-beta.yaml]: gcr

Comment (kawych) [reply custom-metrics-stackdriver-adapter/adapter-beta.yaml]: Fixed.

File: custom-metrics-stackdriver-adapter/adapter-beta.yaml (hunk 1 above)
Comment (kawych) [reply custom-metrics-stackdriver-adapter/adapter-beta.yaml]: No, this needs to be kube-system. AFAIU the RoleBinding needs to be in the same namespaces as the referenced Role, which is kube-system in this case.

**Comment by piosz** [comment]
@sttts could you please take a look into rbac config?

**Comment by piosz** [comment]
LGTM

**Pull request merged**